themes/tec/scss/theme/_main.scss

```diff
@@ -395,8 +395,10 @@ section[rel~="schema:comment"] {
 
 /* Contact page styles */
 
-.contact-hero {
-  img {
+
+.block-tec-contacthero {
```

Comment:
Yeah, exactly. Like, why did we ever want a module just to do that when we could've written basically the same css? Idk...